swan-lake/featured-use-cases/work-with-data-using-queries-in-ballerina.md

```diff
@@ -434,4 +434,268 @@ Countries with highest deaths:[["India",980976],["USA",880976],["Afghanistan",73
 Countries with number of Recovered patients:[["Afghanistan",146084],["USA",43892277],["India",33892279]]
 Found erroneous entries for countries: ["Sri Lanka","India"]
 ```
-   
\ No newline at end of file
+## Group COVID-19 data by continent
+
+### Create the `groupDeathsByContinent` function
+
+To difine a function to group the data by continent and get the total number of deaths by continent, add the code below to the `main.bal` file.
+
+```ballerina
+public function groupDeathsByContinent(table<CovidEntry> dataTable) returns record {|string continent; decimal deaths;|}[] {
+    return from var {continent, deaths} in dataTable
+        group by continent
+        select {continent, deaths: sum(deaths)};
+}
+```
+
+In this code,
+- You use the `group by` clause to group the input. After the `group by` you can find the `grouping-key`, in this example `continent` is the `grouping-key`. The `grouping-key` is used to group the input and each group, the `grouping-key` is unique.
```

Comment:
`grouping-key` is unique or `grouping-key` should be unique.
what is the best one?

Reply:
I think `is unique` is better.